frontend/src/app/pages/main-page/overview/overview.component.ts

```diff
@@ -0,0 +1,36 @@
+import {Component, ElementRef, OnInit, ViewChild} from '@angular/core';
+import {data} from "../../../data";
+import {OverviewService} from "./overview.service";
+import {Subscription} from "rxjs";
+import {BookInterface} from "../../../interfaces/book.interface";
+
+@Component({
+  selector: 'app-overview',
+  templateUrl: './overview.component.html',
+  styleUrls: ['./overview.component.scss']
+})
+export class OverviewComponent implements OnInit {
+
+  private sub$ = new Subscription();
```

Comment:
Symbol '$' is used only for 'Observable' object, but here you use 'Subscription'. Please remove symbol '$' and use other name - 'subscriptions'